### PR TITLE
Update existing ug

### PR DESCRIPTION
**Pull request by ziiqii**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -132,7 +148,7 @@ Examples:
 
 ### Finding persons by name: `find`
 
-Finds persons whose names contain any of the given keywords.
+Finds persons whose names contain any of the given keywords in the displayed person list.
```
Comment (KSSWSept20) [inline docs/UserGuide.md:151]:
```suggestion
Finds persons whose names contain any of the given keywords in the _displayed person list_.
```
I think it's bettter to italicize the _displayed person list_.

File: docs/UserGuide.md [hunk 2]
```diff
@@ -205,33 +223,35 @@ If your changes to the data file makes its format invalid, CulinaryContacts will
...
-**Edit**   | `edit INDEX [n/NAME] [p/PHONE_NUMBER] [e/EMAIL] [a/ADDRESS] [t/TAG]…​`<br> e.g.,`edit 2 n/James Lee e/jameslee@example.com`
-**Find**   | `find KEYWORD [MORE_KEYWORDS]`<br> e.g., `find James Jake`
-**List**   | `list`
-**Help**   | `help`
+## Command Summary
+
+| Command       | Format, Examples                                                                                                                                                     |
+|---------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| **help**      | `help`                                                                                                                                                               |
+| **add**       | `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]…​`<br> e.g., `add n/James Ho p/22224444 e/jamesho@example.com a/123, Clementi Rd, 1234665 t/friend t/colleague` |
```
Comment (zhuuyicheng) [inline docs/UserGuide.md:244]: Just gotta use the `add` command example here

File: docs/UserGuide.md [hunk 3]
```diff
@@ -171,21 +190,20 @@ Format: `delete INDEX`
...
 Examples:
-* `list` followed by `delete 2` deletes the 2nd person in CulinaryContacts.
-* `find Betsy` followed by `delete 1` deletes the 1st person in the results of the `find` command.
+* `list` followed by `delete 2` deletes the 2nd person in the contacts list of CulinaryContacts.
 
-### Clearing all entries : `clear`
+### Clearing all entries: `clear`
 
-Clears all entries from CulinaryContacts.
+Clears all persons and reservations from CulinaryContacts.
```
Comment (zhuuyicheng) [inline docs/UserGuide.md:197]:
```suggestion
Remove all persons and reservations from CulinaryContacts.
```

File: docs/UserGuide.md [hunk 4]
```diff
@@ -150,7 +166,7 @@ Examples:
 
 ### Filtering persons by tag: `filter`
 
-Finds persons that are tagged with all of the given tags.
+Finds persons that are tagged with all the given tags in the displayed person list.
```
Comment (KSSWSept20) [inline docs/UserGuide.md:169]:
```suggestion
Finds persons that are tagged with all the given tags in the _displayed person list_.
```

**Review by zhuuyicheng — APPROVED** [review]
LGTM!

**Review by KSSWSept20 — CHANGES_REQUESTED** [review]

File: docs/UserGuide.md (hunk 1 above)
Comment (ziiqii) [reply docs/UserGuide.md]: fixed

**Review by KSSWSept20 — APPROVED** [review]
LGTM!

**Pull request merged**